dashboard/test/controllers/registrations_controller_test.rb

```diff
@@ -696,117 +668,60 @@ class RegistrationsControllerTest < ActionController::TestCase
     assert_equal EmailPreference::ACCOUNT_SIGN_UP, email_preference[:source]
   end
 
-  test "create new teacher with us ip with opt-in to sharing email with regional partners set share_teacher_email_regional_partner_opt_in value to DateTime value [new sign up flow]" do
```

Comment:
We no longer utilize the `share_teacher_email_regional_partner_opt_in` feature so these tests don't need to be duplicated for the new signup flow (could potentially even be removed for the old flow too)